drivers/modem/gsm_ppp.h

```diff
@@ -0,0 +1,27 @@
+/** @file
+ * @brief Modem socket header file.
+ *
+ * Generic modem socket and packet size implementation for modem context
+ */
+
+/*
+ * Copyright (c) 2019-2020 Foundries.io
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_MODEM_MODEM_GSM_PPP_H_
+#define ZEPHYR_INCLUDE_DRIVERS_MODEM_MODEM_GSM_PPP_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+void restart_gsm_init(void);
```

Comment:
Please rename this to `gsm_init_restart()`

Reply:
addressed